content/docs/1.4/scalers/cron.md

```diff
@@ -0,0 +1,51 @@
++++
+title = "Cron"
+layout = "scaler"
+availability = "v1.0+"
```

Comment:
This is not available as of 1.0, let's move to 2.0
```suggestion
availability = "v2.0+"
```

Reply:
As discussed, this will stay with 1.0

Reply:
yeah, but the version that will introduce this scaler will be most probably 1.5, not 1.0

Reply:
That's correct, updated suggestion.